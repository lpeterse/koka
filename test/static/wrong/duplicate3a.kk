// duplicate constructor
type dup1 { Dup1(:int) }
type dup2 { Dup1(:int) }
