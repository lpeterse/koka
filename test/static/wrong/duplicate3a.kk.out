test/static/wrong/duplicate3a.kk(3,13): error: Constructor static/wrong/duplicate3a/Dup1 is already defined at (2,13)